src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -403,10 +403,10 @@ void CoilCoolingDXCurveFitPerformance::size(EnergyPlus::EnergyPlusData &state)
     if (!state.dataGlobal->SysSizingCalc && this->mySizeFlag) {
         this->normalMode.parentName = this->parentName;
         this->normalMode.size(state);
-        if (this->hasAlternateMode == HVAC::coilEnhancedMode) {
+        if (this->coilMode == HVAC::CoilMode::Enhanced) {
             this->alternateMode.size(state);
         }
-        if (this->hasAlternateMode == HVAC::coilSubcoolReheatMode) {
+        if (this->coilMode == HVAC::CoilMode::SubcoolReheat) {
             this->alternateMode.size(state);
             this->alternateMode2.size(state);
         }
```

Comment:
So here if hasAlternateMode == coilEnhanceMode call one size function. If coilSubcoolReheatMode call 2 size functions. That just feels odd. I digress.

Reply:
Breaking news, EnergyPlus is odd!